### DISCUSSION TITLE
How generate all tailwindcss classes

### DISCUSSION THREAD

**Discussion by jairoblatt** [tailwindlabs/tailwindcss#8147] [Help]
Hello everyone, I need to generate all tailwind classes with my custom theme (I know they are huge CSS classes) to upload to a private cdn.

When I run the build with `content=[/* Empty */]` a message appears `warn - The 'content' option in your Tailwind CSS configuration is missing or empty.`

*ps: i'm working with shopify liquid theme, this is the only way to work with tailwind in preview mode.

How i can force generate all tailwindcss classes?

Thanks.

**Comment by SSHari** [comment]
You should be able to use a regex in a [safelist](https://tailwindcss.com/docs/content-configuration#using-regular-expressions) to handle this. If you set the regex pattern to something like `safelist: [{ pattern: /./ }],` in your `tailwind.config.js` you should be able to generate all the classes.

**Reply by jairoblatt** [reply]
it worked very well, thanks

**Reply by gbrits** [reply]
Guess this doesn't work anymore

**Reply by wongjn** [reply]
It should still work in Tailwind v3. Though do bear in mind this does not generate any arbitrary classes, or any classes with variants.

**Reply by AlonMiz** [reply]
## WARNING

`safelist: [{ pattern: /./ }]` does work.

**BUT**, this can slow down the development time by a large amount. in my case, working with storybook and this config, just made every HMR take 8.5s more than 40ms than it should have.

so use it with caution!
